cost-analyzer/values.yaml

```diff
@@ -1010,6 +1010,9 @@ kubecostAdmissionController:
 costEventsAudit:
   enabled: false
 
+# Instead of a kubecost-analyzer Deployment, you can set it to be a StatefulSet as for volumeClaimTemplates usage and real stateful behaviour
+statefulSet:
+  enabled: false
```

Comment:
Because this setting only impacts the cost-analyzer Deployment, I would prefer to see this under the top-level object `kubecostDeployment`.

Reply:
done ✔️